curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/compare-scopes-of-the-var-and-let-keywords.english.md

```diff
@@ -31,7 +31,7 @@ This exercise is designed to illustrate the difference between how <code>var</co
 ```yml
 tests:
   - text: <code>var</code> does not exist in code.
```

Comment:
@thecodingaviator Since your changing the error message in the testString you might as well change it in the text also.
```suggestion
   - text: <code>var</code> should not exist in code.
```